testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveRedirectionToIceberg.java

```diff
@@ -374,6 +374,20 @@ public void testCommentColumn()
         onTrino().executeQuery("DROP TABLE " + icebergTableName);
     }
 
+    @Test(groups = {HIVE_ICEBERG_REDIRECTIONS, PROFILE_SPECIFIC_TESTS})
```

Comment:
I wanted to do also a test on `io.trino.tests.product.iceberg.TestIcebergRedirectionToHive` class, but I stumbled on `Not an Iceberg table` exception which needs first  https://github.com/trinodb/trino/issues/10245 to be implemented.

Reply:
please add a sibling test case there so that we don't forget to test-cover this when #10245 is done.